docs/components/position.md

```diff
@@ -45,3 +45,40 @@ entity. In the local parent's space, `#child1`'s position would be `0 0 0`.
 The world-space position of `#child2` would be `3 5 7`, by combining the
 position with the parent entity. In the parent's local space, `#child2`'s
 position would be `2 3 4`.
+
+## Updating Position
+
+[object3d]: https://threejs.org/docs/#api/core/Object3D
+[update]: ../introduction/javascript-events-dom-apis.md#updating-a-component-with-setattribute
+[vector]: https://threejs.org/docs/index.html#api/math/Vector3
+
+For performance and ergonomics, we recommend updating position directly via the
+three.js [Object3D][object3d] `.position` [Vector3][vector] versus [via
```

Comment:
I don't think they use it since those events are throttled. For position/rotation/scale/visible updates, a tick handler checking for changes is preferred. https://github.com/networked-aframe/networked-aframe/search?utf8=✓&q=componentchanged&type= 

I can add a note though

Reply:
Worth adding the note but IMO this will be a huge benefit for physics where many things are constantly in motion, and the events are more overhead. Still planning to put together a stress test but haven't hand the chance yet. :/

Reply:
@haydenjameslee does prescribing the direct modification of object3D instead of doing `setAttribute` will affect `networked-aframe`? I know you were doing experiments last week hooking into events.

Reply:
The lerp component of networked-aframe used to use the `componentchanged` event, however to support A-Frame 0.6 I changed it to poll in the tick: https://github.com/haydenjameslee/aframe-lerp-component/commit/e593f75432ffc7a1717da933c0308673cd1e1f56#diff-168726dbe96b3ce427e7fedce31bb0bc

So this change should be fine.

Reply:
Cool! Thanks! 👍